File: Typescript/src/getSneakyNumbers.ts
export function getSneakyNumbers(nums: number[]): number[] {
	const count = new Map<number, number>();
	let ret = [];
	for (const num of nums) {
		if (1 === count.get(num)) {
			ret.push(num);
		} else {
			count.set(num, 1);
		}
	}
	return ret;
};

File: Dart/lib/getSneakyNumbers.dart
class Solution {
  List<int> getSneakyNumbers(List<int> nums) {
    var cnt = List.filled(nums.length, 0);
    List<int> sneakyNumbers = [];
    for (int num in nums) {
      if (cnt[num] > 0) {
        sneakyNumbers.add(num);
      }
      cnt[num]++;
    }
    return sneakyNumbers;
  }
}
